### PR TITLE
Update dependencies for roboRIO image v4

### LINKED ISSUE #68: Update to new roboRIO version
For this update wpilib and ctre dependencies need to be updated as well.

### PR DESCRIPTION
**Pull request by adamv17**
closes #68

### CODE REVIEW

**Review by barel13 — CHANGES_REQUESTED** [review]
I'm sorry, I cant approve it without checking the consequences of those changes. Maybe try next year, God luck in the future.

**Review by barel13 — APPROVED** [review]

**Review by katzuv — APPROVED** [review]
Great! Don't forget to merge `main` into `develop` after this PR is merged.
Also, remember that the convention for branch names is using dashes instead of underscores.

**Pull request merged**
